manifests/l/lazygamereviews/KEmulator/1.0.3/lazygamereviews.KEmulator.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: lazygamereviews.KEmulator
+PackageVersion: 1.0.3
+Installers:
+- InstallerUrl: https://archive.org/download/KEmulator103/KEmulator%201.0.3%20modded.zip

```

Comment:
Archive.org's TOS only allow for scholarship and research purposes only. Winget does not fall into the category of Scholarship and/or Research, so we cannot use them as a source